src/components/Stages/genericStages.js

```diff
@@ -0,0 +1,99 @@
+import main from "../../images/stages/main.png";
+import district from "../../images/stages/district.png";
+// import soundr from '../../images/stages/soundR.png';
+import truband from "../../images/stages/truband.png";
+import rare from "../../images/stages/rare.png";
+import soundr from "../../images/stages/soundr.png";
+import nuggs from "../../images/stages/nuggs.png";
+import azul from "../../images/stages/azul.jpg";
+import underground from "../../images/stages/underground.jpg";
+import portaloo from "../../images/stages/Hour-Glass.png";
+import moda from "../../images/stages/moda.jpg";
+import jazz from "../../images/stages/jazz.png";
+import techno from "../../images/stages/techno.png";
+import rage from "../../images/stages/rage.jpg";
+import ravers from "../../images/stages/rr.png";
+
+export default [
+  {
+    name: "Random Ravers",
+    href: "https://play.decentraland.org/?position=-54%2C60&realm=marvel&island=Ic9b0",
```

Comment:
The site URL for play changed to:
```suggestion
    href: "https://decentraland.org/play?position=-54%2C60&realm=marvel&island=Ic9b0",
```
Would you mind updating it?